string name
geomtwo/Versor bearing
---
float64 beta
float64 next_access
